src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/SetsRequiredMembersAttribute.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
```

Comment:
These #nullable enables aren't necessary as we enable it at the project level (and there's nothing in the APIs that's impacted).

Reply:
Ah. I was just copying from other attributes in the namespace.